### DISCUSSION TITLE
The old man needs help can someone help him

### DISCUSSION THREAD

**Discussion by don-recardo** [ElectroTechnique/electrotechnique.github.io#66] [Software Issues]
Hi I'm asking for help because at 72 years old things dont make as much sense to me now as they do with you youngsters
( Think of old dogs and new tricks)

I built the Electrotechnique Tsynth from the kit and loaded the v2.31 hex file into my teensy 4.1 and to my great suprise it worked first go  . 
It soundes great , all the controls work and LEDs light up when they should however ( theres always a however)
the display has the dreaded slanty text . I looked online and its seems this is quite common and there is an easy fix .
In another post on here I found ..... The solution is to modify the code that initialises the display (in ST7735Display.h). The following lines can be changed:

  tft.initR(INITR_GREENTAB); //INITR_BLACKTAB 
  tft.invertDisplay(true); // false

I downloaded the Tsynth teensy 4.1 master file on github and in it opened TSynth.ino in the arduino ide . Across the top were all the extra files with *.H extention  I found the ST7735Display.h  and changed the two lines as the fix stated , but then I am stuck . I have no idea how to get the whole thing back into one Hex file  like V2.31 hex  that I could then upload using Teeensy Loader like before.

I tried clicking verify in the IDE but it gave an error  AUDIO USB does not name a type even though the only thing I have changed were the two lines it said I should alter.
So now I'm stuck
Please please pretty please could someone who knows what they are doing modify the two lines that need changing and remake it into a single V2.31.hex or V2.31fixed.hex file and post it back here because I dont have a chance of me working it out for myself  and it seems a pity to have a great synth ruined just because of the display problem

So , can you help this old phart?

Regards
Don

**Comment by don-recardo** [comment]
If anyone has the same sloping text display here is the fixed hex file which Iain from 
ElectroTechnique has kindly allowed me to post

Don

https://www.dropbox.com/s/uix2ky5z6r2d95f/TSynth-fixed-display.ino.hex?dl=0

**Reply by Electrotechnique** [reply]
Hello, sorry I'm a bit late to this. Glad you solved it.
